module/entries/MassEdits/ChangesTo-Subconnections-Sample.ps1

```diff
@@ -0,0 +1,76 @@
+#requires -version 4
+<#
+.SYNOPSIS
+  Mass edit SubConnections (parent/childs) in RDM
+
+.DESCRIPTION
+  Mass edit SubConnections in RDM by Un-Parenting SubConnections by ConnectionType, 
+  then make the needed edits to the entry and then re-parenting the entry
+
+.PARAMETER <Parameter_Name>
+
+.INPUTS
+  Confirmation will be required for EACH Un-Parent
+
+.OUTPUTS
+  None
+
+.NOTES
+  Version:        1.0
+  Author:         Eric St-Martin
+  Creation Date:  05/05/2021
+  Purpose/Change: Initial script development
+.EXAMPLE
+
+#>
+
+#-------------------------------[Initialisations]------------------------------
+
+#Set Error Action to Silently Continue
+$ErrorActionPreference = 'SilentlyContinue'
+
+#Import Modules & Snap-ins
+Import-Module RemoteDesktopManager.PowerShellModule

```

Comment:
I would change to:

if ( ! (Get-Module RemoteDesktopManager.PowerShellModule)) {
Import-Module "${env:ProgramFiles(x86)}\Devolutions\Remote Desktop Manager\RemoteDesktopManager.PowerShellModule.psd1"
}